cmd/ncproxy/run.go

```diff
@@ -35,13 +36,15 @@ type nodeNetSvcConn struct {
 var (
 	// Global mapping of network namespace ID to shim compute agent ttrpc service.
 	containerIDToShim = make(map[string]computeagent.ComputeAgentService)
```

Comment:
Would be good to have another PR to get rid of these globals. Currently if you tried to host two ncproxy services in a single process they would have conflicting access to these globals.